scoala-online/main
# Feat/#30 create module page

### This pull request solves issue:

Closes: #30 

### Which app is this issue related to?


### Types of changes


### Wiki Updates


### Testing checklist


#### Security Filters - **_For API Only_**

**_Please make sure to add the respective Security Filters before submitting a PR_**


#### Compatibility tests - **_For E-Learning React App Only_**


### What changes have been done within this issue?

Within this issue, I have added two components consisting of a sidebar component and a container including videos both encapsulated in the main component `Module`.

**Static Data**
For the sake of working without using Axios , I added static data in the `app/components` folder.
1. `VideoContent` includes examples of video data used to display the videos.
2. `SidebarData` includes strings and list of strings with a title attribute representing the lectures on the sidebar.

**Components**
1. The `VIdeo` component constructs a video using data from `VideoContent`
2. The `VideoRow` component containing multiple `Video` components
3. The `Sidebar.js` and `Sidebar.css` containing the `SidebarData` entries
4. The `Module` encapsulating all components 

### How should this be manually tested?

Run the command `nx serve e-learning-app` and try shrinking, expanding and clicking all the components in order to verify all the expected functionalities required [here](https://www.figma.com/file/RAVu7vo2GNkr0mCTh6TkDc/ScoalaOnline-UX?node-id=148%3A65)
